### PR TITLE
#2515: make DNA raw reads filter wider

### PR DESCRIPTION
**Pull request by samrichca**
# Description of changes

- add CSS to make DNA raw reads filter wider
- requires Uikit change to accept a className prop https://github.com/icgc-argo/uikit/pull/106

## Type of Change

- [ ] Bug
- [ ] Dependency updates
- [ ] Feature
- [ ] Refactoring
- [ ] Release candidate
- [x] Styling
- [ ] Testing
- [ ] Other (please describe)

## Checklist before requesting review

- Design (select one):
  - [x] Matches design:
    - component sizes, spacing, and styles
    - font size, weight, colour
    - spelling has been double checked
  - [ ] No design provided, screenshot of changes included in PR
  - [ ] No changes to UI
- Back-end (select one):
  - [ ] Back-end PRs merged, tested on develop, and linked in this PR
  - [x] No back-end changes
- [x] Manually tested changes
- [x] Added copyrights to new files
- [x] Connected ticket to PR

### CODE REVIEW

**Comment by daniel-cy-lu** [comment]
Small styling change. Should I wait for the UIkit PR to merge before I approve this one? @samrichca

**Comment by samrichca** [comment]
@daniel-cy-lu sure, I made both PRs at the same time for context for reviewers.

**Review by daniel-cy-lu — APPROVED** [review]
minor css update to DNA raw reads filter.

File: components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx [hunk 1]
```diff
@@ -257,6 +259,12 @@ const DonorSummaryTable = ({
           panelRef={panelRef}
           handleBlur={handleBlur}
           active={active}
+          css={css`
+            ${['DNA Raw Reads Status'].includes(panelLegend) &&
+            `
+              width: 275px;
+            `}
+          `}
```
Comment (joneubank) [inline components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx:267]: This doesn't make sense to me just reading the code... can you add some comments to describe the purpose of "panelLegend" and why this is needed?

**Review by joneubank — APPROVED** [review]
All fine but could use some more descriptive comments probably

File: components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx (hunk 1 above)
Comment (samrichca) [reply components/pages/submission-system/program-dashboard/DonorDataSummary/DonorSummaryTable.tsx:267]: sure

**Review by joneubank — APPROVED** [review]

**Pull request merged**